### DISCUSSION TITLE
Change height of navbar

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [saadeghi/daisyui#1981] [Q&A] [answered]
Hi,

Is it possible to change the height of the navbar. I want that for mobile devices it should have a height of h-8.

Any help is much appreciated

**Comment by saadeghi** [comment, marked as answer]
`navbar` also has a `min-height`  
So you can add `min-h-0 h-8` class names to `navbar` to change the height as you want

**Reply by MuslemRahimi** [reply]
thank you very much. Works fine!
